src/components/PurchaseOrder/POForm.js

```diff
@@ -178,11 +179,15 @@ class POForm extends Component {
   }
 
   onChangeTemplate = (e, value) => {
-    const { change, dispatch, parentResources, stripes } = this.props;
+    const { change, dispatch, reset, parentResources, stripes } = this.props;
     const templateValue = getOrderTemplateValue(parentResources, value);
     let form = get(stripes.store.getState(), 'form', {});
+    const { FormPO: { values: { poNumber } } } = form;
 
+    dispatch(reset('form'));
```

Comment:
@Vladyslav-Velytskyi are you sure it should be `'form'`?

Reply:
@aliaksei-chumakou not really, I rechecked, `'form'` could be omit, thanks!